components/infieldbutton/index.css

```diff
@@ -391,6 +398,14 @@ governing permissions and limitations under the License.
 			--mod-infield-button-stacked-border-radius-reset,
 			var(--spectrum-infield-button-stacked-border-radius-reset)
 		);
+		border-end-end-radius: var(
+			--mod-infield-button-stacked-bottom-border-radius-end-end,
+			var(--spectrum-infield-button-stacked-bottom-border-radius-end-end)
+		);
```

Comment:
Rather than defining `--spectrum-infield-button-stacked-bottom-border-radius-end-end`, could the value of this just be:
```
var(
	--mod-infield-button-stacked-bottom-border-radius-end-end,
	var(--mod-infield-button-border-radius, var(--spectrum-infield-button-border-radius))
);
```
The same question for `--spectrum-infield-button-stacked-bottom-border-block-end-width`.

Reply:
Oh yeah, this is way better. I'll update, thank you!